### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by yixinglu**
as title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,44 +80,63 @@ Date: Fri, 02 Apr 2021 08:49:18 GMT
 
 Notice:
 
-The response data nsid "5e18fa40-5343-422f-84e3-e7f9cad6b735" is encoded by HMAC-SH256 encryption algorithm，so it's not the same as what you get from a cookie.
+The response data nsid `5e18fa40-5343-422f-84e3-e7f9cad6b735` is encoded by HMAC-SH256 encryption algorithm, so it's not the same as what you get from a cookie.
 If you connect the graphd service succeed, remember to save the *NSID*, it's important for the *exec api* to execute nGQL.
```
Comment (Aiee) [inline README.md:84]: connect the graphd service successfully

File: README.md [hunk 2]
```diff
@@ -69,44 +80,63 @@ Date: Fri, 02 Apr 2021 08:49:18 GMT
...
 If you restart the gateway server, all authenticated session will be lost, please noticed.
```
Comment (Aiee) [inline README.md:85]: please be aware of this.

File: README.md (hunk 1 above)
Comment (yixinglu) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (yixinglu) [reply README.md]: Done

**Review by Aiee — APPROVED** [review]

**Pull request merged**